languagetool-language-modules/de/src/main/java/org/languagetool/rules/de/GermanSpellerRule.java

```diff
@@ -115,6 +115,8 @@ public class GermanSpellerRule extends CompoundAwareHunspellRule {
     put("Kruks", "Krux");
     put("Filterbubble", "Filterblase");
     put("Filterbubbles", "Filterblasen");
+    put("gunsten", "Gunsten");
+    put("ungunsten", "Ungunsten");
```

Comment:
This will just make "Gunsten" the first suggestion, not the only one. You will have to use the `topMatch` approach further below

Reply:
Thank you! I also added this to the confluence file overview page.